eva-ingest-remapped-variants/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it possible at all to have the tool for extraction `eva-accession-remapping` and ingestion `eva-ingest-remapped-variants` being co-located ? 
Or at least being named in a consistent way? Happy for `eva-accession-remapping` to be renamed if required.

Reply:
I renamed them to `eva-remapping-ingest` and `eva-remapping-get-source`